1c.cpp

```diff
@@ -0,0 +1,35 @@
+#include <algorithm>
+#include <list>
+#include <iostream>
+
+using namespace std;
+int main()
+{
+    list<int> l;
+    int i;
+    int j;
+    cout<<"Введите количество элементов списка\n";
+    int imax;
+    cin>>imax;
+    cout<<"Введите элементы списка\n";
+    for(i = 0 ;i < imax;i++){
+        std::cin>>j;
+        l.push_back(j);
+    }
+    l.sort();
+    cout<<"Производится сортировка:\n";
```

Comment:
The message is not informative, it would be sent to the standard output only after the actual sort had already happened.